contributors/design-proposals/bootstrap-discovery.md

```diff
@@ -220,16 +219,27 @@ Only one of `--discovery-url`, `--discovery-file` or `--discovery-token` can be
 
 ## Implementation Details
 
-Our documentations (and output from `kubeadm`) should stress to users that when the token is configured for authenitication and used for TLS bootstrap (using `--insecure-experimental-approve-all-kubelet-csrs-for-group`) it is essentially a root password on the cluster and should be protected as such.  Users should set a TTL to limit this risk.  Or, after the cluster is up and running, users should delete the token using `kubeadm token delete`.
+Our documentations (and output from `kubeadm`) should stress to users that when the token is configured for authentication and used for TLS bootstrap is a pretty powerful credential due to that any person with access to it can claim to be a node.
+The highest risk regarding being able to claim a credential in the `system:nodes` group is that it can read all Secrets in the cluster, which basically compromises the cluster, until the [Node Authorizer](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/kubelet-authorizer.md) is fully implemented.
```

Comment:
even once the node authorizer is in place, any credential that can be used to obtain credentials for any node (which the bootstrapper can) can still access secrets used by any pod